### ISSUE TITLE
{SIGNUP_GDPR_INFO} and bootstrap5 - html markup is not correct

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#4638]
## Bug Description  
info is separated to more lines
 
![image](https://user-images.githubusercontent.com/5429548/147007251-bc48d6b5-c0ab-4e05-8c9d-02ed25a75b02.png)

## How to Reproduce
1. Use **core boostrap5 theme**
2. Click on signup page , add GDPR settings
 
Problem is in using row class in this:
`$SIGNUP_WRAPPER['SIGNUP_GDPR_INFO']             = "<div class='form-group row m-2 text-center'>{---}</div> ";`
 
the row has different behavior in bootstrap5  (flex is added)
maybe there is another fix, but removing the class row helped

![image](https://user-images.githubusercontent.com/5429548/147007475-b68bdc84-bc53-4129-9fe1-e74ba961095b.png)

**Comment by CaMer0n** [comment]
@Jimmi08 I believe all the text is meant to be centered.

**Comment by Jimmi08** [comment]
@CaMer0n yes, it should be centered, but flex in BT5 is splitting text to more than one line (the long line is ok but now it looks like 4 lines - link is on new line) - because class row. 
- not sure now
